integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/conditionalRouter/ConditionalRouterIntegrationTest.java

```diff
@@ -223,13 +221,10 @@ public void conditionalRouterMediatorWithDynamicSequenceTest() throws Exception
         // Add dynamic sequence to WSO2registry    config/filters/dynamic_seq1
         if (ArrayUtils.contains(sequenceAdminServiceClient.getDynamicSequences(), "conf:/filters/dynamic_seq1")) {
             sequenceAdminServiceClient.deleteDynamicSequence("conf:/filters/dynamic_seq1");
-            for (int i = 0; i < 5; i++) {
-                if (!ArrayUtils
-                        .contains(sequenceAdminServiceClient.getDynamicSequences(), "conf:/filters/dynamic_seq1")) {
-                    break;
-                }
-                Thread.sleep(500);
-            }
+
+            Awaitility.await().pollInterval(500, TimeUnit.MILLISECONDS).atMost(
+                    60, TimeUnit.SECONDS).until(dynamicSequenceExists(sequenceAdminServiceClient.
+                    getDynamicSequences(), "conf:/filters/dynamic_seq1"));
```

Comment:
Define the configuration path as a constant.

Reply:
Fixed in https://github.com/wso2/product-ei/pull/3665